src/pudl/helpers.py

```diff
@@ -2327,3 +2332,152 @@ def standardize_phone_column(df: pd.DataFrame, columns: list[str]) -> pd.DataFra
         df[column] = df[column].mask(invalid_mask, pd.NA)
 
     return df
+
+
+class ParquetData(BaseModel):
```

Comment:
I now have this class as well as a few helper functions that either take it as a parameter or return one/more instance of `ParquetData`.